src/components/Trade/TradeInput.tsx

```diff
@@ -220,6 +206,18 @@ export const TradeInput = ({ history }: RouterProps) => {
       return 'common.insufficientAmountForGas'
     }
 
+    if (Boolean(quoteError)) {
```

Comment:
Out of curiosity @gomesalexandre is there any reason you prefer `Boolean` over `!!`?